### PR TITLE
Sanity check for pill creation (0u pills)

### PR DESCRIPTION
**Pull request by Birdtalon**
A check to see if the reagents datum actually contains any reagents before creating a pill.

🆑 Birdtalon
fix: Prevents creation of 0u pills with Chem Master
/🆑

### CODE REVIEW

File: code/modules/reagents/chemistry/machinery/chem_master.dm [hunk 1]
```diff
@@ -202,6 +202,8 @@
 					return
 				name = reject_bad_text(name)
 				while(count--)
+					if(!LAZYLEN(reagents.reagent_list))
```
Comment (Birdtalon) [inline code/modules/reagents/chemistry/machinery/chem_master.dm]: I couldn't find a dedicated proc for checking if the reagent datum actually contained reagents. If I missed it please point it out.

Comment (FlattestGuitar) [reply code/modules/reagents/chemistry/machinery/chem_master.dm]: `reagents.volume > 0`?

Comment (Birdtalon) [reply code/modules/reagents/chemistry/machinery/chem_master.dm]: Actually good point, didn't think of that.

**Comment by Plotron** [comment]
I'm not sure that's the best way of handling this. But I guess that even placebo is just sugar!

File: code/modules/reagents/chemistry/machinery/chem_master.dm [hunk 2]
```diff
@@ -202,6 +202,8 @@
 					return
 				name = reject_bad_text(name)
 				while(count--)
+					if(reagents.total_volume <= 0)
```
Comment (Crazylemon64) [inline code/modules/reagents/chemistry/machinery/chem_master.dm]: Perhaps an error message could be displayed to the user here?

Comment (Birdtalon) [reply code/modules/reagents/chemistry/machinery/chem_master.dm]: Sounds good. Will add that in soon.

**Pull request merged**